### ISSUE TITLE
U2F over Bluetooth

### ISSUE THREAD

**Issue by Kami-no** [flipperdevices/flipperzero-firmware#2513]
### Description of the feature you're suggesting.

It would be great to have BT support because wires are boring and MacBook has no NFC.

**Comment by evgfilim1** [comment]
I'll try to implement it. @skotopes can you assign it to me?

If you don't hear from me anything during about a month, it means I'm too busy and someone can take the task.

**Comment by skotopes** [comment]
We'll wait for your PR ;-)

**Comment by xnohat** [comment]
Can we implement this feature @skotopes  ? U2F over bluetooth a great because on market only a fews option.

**Comment by skotopes** [comment]
@xnohat we can implement anything, but that doesn't mean that we should.

In this specific case there are some concerns:
- U2F is superseded by FIDO. That makes implementing FIDO higher priority task.
- Bluetooth in our specific case means BLE, before starting U2F/FIDO over BLE I'd like to know how many systems are on the market and how well developed technology is.

As far as I understand whole thing is a very niche(same as U2F over NFC), which makes is low priority.

**Comment by huan** [comment]
I'm so looking forward to using U2F over Bluetooth on my Mac! (I just got my Flipper this week, and it's so good that far more than my expectation!)

**Comment by WestonGFX** [comment]
This is something I would like to do U2F over Bluetooth on MacOS as well with my F0.

**Comment by lzlrd** [comment]
https://github.com/fmeum/WearAuthn might be a good reference, supports FIDO[2] over Bluetooth and NFC (https://github.com/flipperdevices/flipperzero-firmware/issues/1565).

**Comment by rouing** [comment]
I use FIDO over BLE and NFC regularly. More so NFC as it's our way of doing authentication into AD Domains. It's easier than having people try to plugin their key into their laptops (surfaces) whenever more than likely they have no USB ports left on the laptop or dock. Just putting my 2c
